packages/paste-core/components/form/src/select/Select.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import css from '@styled-system/css';
+import ChevronDown from '@twilio-paste/icons/svg/ChevronDown.svg';
```

Comment:
We use a library to convert our icon svgs into react components. Changing the import path to: `@twilio-paste/icons/esm/ChevronDownIcon` will point to the react component version.

Reply:
Is that compatible with background image? I think I need to use an svg for the css background, or is there a way to use the icon?

Reply:
Hmm. Right. That might cause some issues. Let me think about that one a bit.